spec/mailers/comment_mailer_spec.rb

```diff
@@ -2,6 +2,8 @@
   include MailerSpecHelper
   include EnvVarSpecHelper
 
+  let(:user) { create(:user) }
```

Comment:
I started doing that, then noticed there is a dependency on `user` in the bunch of `let` statements in the first test.